frontend/src/components/common/Grid/components/Loader/index.tsx

```diff
@@ -1,11 +1,18 @@
 import React from "react";
 import { LoadingIndicator } from "@czi-sds/components";
-import { Loader as GridLoader } from "./style";
 
-export default function Loader(): JSX.Element {
+interface Props {
+  className?: string;
+  sdsStyle?: "minimal" | "tag";
+}
+
+export default function Loader({
```

Comment:
For reuse, the Loader component is updated so that the `LoadingIndicator` can take either `sdsStyle` "tag" or "minimal" as well as be styled - should it be required.
In making this changes, the loading indicator used by both Collections and Datasets index is updated to a styled component (see `frontend/src/components/common/Grid/components/Loader/style.ts`).